Source/WebKit/WebProcess/WebPage/wpe/WebPageWPE.cpp

```diff
@@ -48,38 +48,34 @@ bool WebPage::platformCanHandleRequest(const ResourceRequest&)
 
 bool WebPage::hoverSupportedByPrimaryPointingDevice() const
 {
-#if ENABLE(TOUCH_EVENTS)
-    return !screenIsTouchPrimaryInputDevice();
-#else
-    return true;
-#endif
+    // For now, there's no concept of primary vs any
+    return WebProcess::singleton().availableInputTypes().contains(AvailableInputTypes::Mouse);
```

Comment:
I think we could use a simple approach for now like, if there's a mouse that's the primary.